### DISCUSSION TITLE
Runtime Error : unexpected EOF, expected 68801193 more bytes. The file might be corrupted

### DISCUSSION THREAD

**Discussion by ghoshsdeep** [open-mmlab/mmsegmentation#2183] [Q&A]
Hello,
I was following the docs https://github.com/open-mmlab/mmsegmentation/blob/master/docs/en/get_started.md#installation
to build the target codebase for mmsegmentation
After installing the prerequisites and cloning the repo ,when I run this command 
python demo/image_demo.py demo/demo.png configs/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes.py pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth --device cuda:0 --out-file result.jpg
I get a runtime error
![Screenshot from 2022-10-13 01-48-26](https://user-images.githubusercontent.com/112154951/195440404-f0e0b06d-321d-4e5c-b73e-b13f7349bc34.png)

Do you have an idea why this might be happening ?

**Comment by MeowZheng** [comment]
I think this file  pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth might be broken, please download a new one.

**Comment by ghoshsdeep** [comment]
Yes,it is working fine now Thank you so much for attending to the query in such a short time.I will close this issue
